CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # SUNDIALS Changelog
 
+## Changes to SUNDIALS in release 7.0.0
+
+The previously deprecated types `realtype` and `booleantype` were removed from `sundials_types.h`.
+Users should use `sunrealtype` and `sunbooleantype` instead, but the old names for the types
+can be used by including the header file `sundials_types_old.h`.
```

Comment:
I think we'd decided on naming this `sundials_types_deprecated.h`. Also slight re-wording to not suggest users use these types.
```suggestion
The previously deprecated types `realtype` and `booleantype` were removed from `sundials_types.h`
and replaced with `sunrealtype` and `sunbooleantype`. The deprecated names for these types
can be used by including the header file `sundials_types_deprecated.h` but will be fully removed in the
next major release.
```

Reply:
Also need to add this entry to the change log in the package docs